### PR TITLE
Make specs more robust

### PR DESCRIPTION
**Pull request by kevgo**
@charlierudolph @allewun @ricmatsui 

After spending a week working on the upcoming Git Town website, and a short stint helping Alex Peachey getting  https://github.com/originate/psapi off the ground, back to Git Town!

This PRs is extracted out of #483, since its unrelated to the changes made there. 
It makes specs more robust.

blocks #483

### CODE REVIEW

File: features/git-new-pull-request/github.feature [hunk 1]
```diff
@@ -5,7 +5,7 @@ Feature: git-new-pull-request when origin is on GitHub
   So that I have more time for coding the next feature instead of wasting it with process boilerplate.
 
 
-  Scenario Outline: result
+  Scenario Outline: creating pull-requests
```
Comment (charlierudolph) [inline features/git-new-pull-request/github.feature]: This seems unrelated and the other feature files should probably be updated in a similar way. Would be good to do in a separate PR.

**Comment by charlierudolph** [comment]
:ship: after the comment is addressed

File: features/git-new-pull-request/github.feature (hunk 1 above)
Comment (kevgo) [inline features/git-new-pull-request/github.feature]: Okay, doing this change separately.
